packages/wrangler/src/__tests__/jest.setup.ts

```diff
@@ -96,7 +95,9 @@ jest.mock("../cfetch/internal");
 	jest.requireActual("../cfetch/internal")["fetchDashboardScript"]
 );
 (fetchInternal as jest.Mock).mockImplementation(mockFetchInternal);
-(fetchKVGetValue as jest.Mock).mockImplementation(mockFetchKVGetValue);
+(fetchKVGetValue as jest.Mock).mockImplementation(
+	jest.requireActual("../cfetch/internal").fetchKVGetValue
+);
```

Comment:
Just remove this now the mock is not being used.

Reply:
The whole internal is still being mocked, removing all the mocking is the last step.